qstnrs/qstnrs/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import patterns, include, url
+from django.contrib import admin
+
+admin.autodiscover()
+
+urlpatterns = patterns(
+    '',
+    url(r'^qstnrs/', include('questionnaire.urls')),
+    url(r'^admin/', include(admin.site.urls)),
```

Comment:
You can check out Projects vs. apps -> https://docs.djangoproject.com/en/1.8/intro/tutorial01/#creating-models